lib/pbench/server/indexing_tarballs.py

```diff
@@ -329,7 +329,7 @@ def sighup_handler(*args):
 
                             try:
                                 dataset = Dataset.attach(
-                                    path=path, state=States.INDEXING
+                                    name=Dataset.stem(path), state=States.INDEXING
```

Comment:
I'm having trouble connecting all the dots...prior to this change, I _think_ the "name" of a dataset was only the stem of the filename; but I think that `Dataset.stem(path)` removes only the suffix and not the parent directory path...is that a problem?

Reply:
The `Dataset` name is the `Path`-speak "stem", which is `.name` - `.suffix`. But the `Path.stem` only removes a single suffix, and our standard tarball format has two.

So `Dataset.stem` removes the `.tar.xz` suffix from the `.name` of the tarball `Path`... e.g., "xyzzy" from `/var/tmp/xyzzy.tar.xz`.

Reply:
OK, nevermind:  I finally tumbled to the fact that in `stem()` we're _also_ using `.name`.  We're all good here.